### PR TITLE
AC_CustomControl: correct error due to changes in PID interface

### PR DESCRIPTION
**Pull request by xianglunkai**
correct error due to changes in PID interface!

### CODE REVIEW

**Comment by peterbarker** [comment]
What error where you encountering?

We have an autotest which is supposed to make sure this keeps working, and it currently passes (`./Tools/autotest/autotest.py build.Copter test.Copter.CustomController`).  Perhaps we need to improve this autotest so whatever you've found here doesn't happen again?

File: libraries/AC_CustomControl/AC_CustomControl_PID.cpp [hunk 1]
```diff
@@ -289,9 +290,9 @@ Vector3f AC_CustomControl_PID::update()
     // run rate controller
     Vector3f gyro_latest = _ahrs->get_gyro_latest();
     Vector3f motor_out;
-    motor_out.x = _pid_atti_rate_roll.update_all(target_rate[0], gyro_latest[0], 1);
-    motor_out.y = _pid_atti_rate_pitch.update_all(target_rate[1], gyro_latest[1], 1);
-    motor_out.z = _pid_atti_rate_yaw.update_all(target_rate[2], gyro_latest[2], 1);
+    motor_out.x = _pid_atti_rate_roll.update_all(target_rate[0], gyro_latest[0], _dt, true);
```
Comment (peterbarker) [inline libraries/AC_CustomControl/AC_CustomControl_PID.cpp:293]: IIUC _dt is correct here, but "limit" should really be false as we're not hitting limits.

**Review by peterbarker — APPROVED** [review]

File: libraries/AC_CustomControl/AC_CustomControl_PID.cpp (hunk 1 above)
Comment (xianglunkai) [reply libraries/AC_CustomControl/AC_CustomControl_PID.cpp:293]: Thank you very much! I will fix it as soon as possible.

**Comment by xianglunkai** [comment]
@peterbarker 
May I ask how to fix the issue of check failure? Please!

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**